### ISSUE TITLE
Printer no redirection Win11 ver. 10.0.22621.3447

### ISSUE THREAD

**Issue by ibanvaqe** [stascorp/rdpwrap#2623]
I have seen that since windows 11 when installing RDPWRAP, the redirection of printers that use the easy print native driver stops working and only works those that have client/server driver.

I have seen other threads about windows 10 but the only solution was to put the library of an old version for the correct functioning of the printer redirection. 

Has anyone tried to fix it in win 11? 

Thanks in advance

**Comment by loyejaotdiqr47123** [comment]
`sc config TermService type= own`
Try running this,then reboot

**Comment by ibanvaqe** [comment]
> `sc config TermService type= own` Try running this,then reboot

I have tried several times but without success :(

**Comment by ibanvaqe** [comment]
I have managed to get it to work. For it I had to modify this value of the hook to 0 so that it does not execute it.

[10.0.22621.3447]
LocalOnlyPatch.x64=1
LocalOnlyOffset.x64=9D661
LocalOnlyCode.x64=jmpshort
SingleUserPatch.x64=1
SingleUserOffset.x64=181AC
SingleUserCode.x64=Zero
DefPolicyPatch.x64=1
DefPolicyOffset.x64=1C2E5
DefPolicyCode.x64=CDefPolicy_Query_eax_rcx
**SLInitHook.x64=0**
SLInitOffset.x64=29ED0
SLInitFunc.x64=New_CSLQuery_Initialize

My doubt is if this hook apart from the printers issue, works in more functionalities, making other sections not to work. 

I will test in the lab to see if other functionalities are still working.
